server/schemas/schemaModel.js

```diff
@@ -28,32 +28,26 @@ module.exports = {
         //console.debug("findOneAsync args:", arguments);
         if (schema) {
           error = {message: "Schema already exists"};
-          helpers.errorHandler(error, request, response, next);
-          return;
+          next(error, false);
         } else {
-          Model.createAsync({
+          return Model.createAsync({
```

Comment:
Interesting, I'm not seeing it when I grep the bluebird module folder or in the [project](https://github.com/petkaantonov/bluebird) page. Could you point me to the documentation?